functests/sctp/sctp.go

```diff
@@ -25,6 +25,7 @@ import (
 const mcYaml = "../sctp/sctp_module_mc.yaml"
 const hostnameLabel = "kubernetes.io/hostname"
 const testNamespace = "sctptest"
+const defaultNamespace = "default"
```

Comment:
what's special about the default namespace here?

usually we use custom namespaces in these tests because it makes it easier to cleanup afterwards.

Reply:
I still need to re-validate it against a fresh cluster (I am starting one right now), but I am almost sure that sctp is not working under the default ns. That's why I added this test.